docs/hacktoberfest/index.html

```diff
@@ -5,14 +5,46 @@
       <meta charset="utf-8"/>
       <meta name="viewport" content="width=device-width, initial-scale=1.0">
       <meta name="description" content="Evento destinado a incentivar a comunidade open source." />
-      <meta name="theme-color" content="#387ca5">
       <link rel="stylesheet" href="./css/styles.css" />
-      <link rel="icon" href="./img/favicon.ico">
+      <link rel="icon" href="./assets/img/favicon.ico">
+      <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.9.0/css/all.css">
   </head>
   <body>
-    <div id="content">
-      <img id="logo" src="./img/logo.svg">
-      <div id="contador"></div>
-    </div>
+    <header>
+      <a  href="https://donorbox.org/hacktoberfest-opendevufcg-2019" target="_blank">
+        <div class="donate-btn">Doar</div>
+      </a>
+    </header>
+    <main id="content">
+      <img id="logo" src="./assets/img/logo.svg">
+      <div id="counter">
+        <div>
+          <span id="days"></span>
+          <span class="counter-label">Dias</span>
+        </div>
+        <div class="dots">:</div>
+        <div>
+          <span id="hours"></span>
+          <span class="counter-label">Horas</span>
+        </div>
+        <div class="dots">:</div>
+        <div>
+          <span id="minutes"></span>
+          <span class="counter-label">Minutos</span>
+        </div>
+        <div class="dots">:</div>
+        <div>
+          <span id="seconds"></span>
+          <span class="counter-label">Segundos</span>
+        </div>
+      </div>
+      <div id='social-medias'>
+          <a href="http://twitter.com/opendevufcg" target="_blank"><i class="fab fa-twitter"></i></a>
+          <a href="https://www.instagram.com/opendevufcg/" target="_blank"><i class="fab fa-instagram"></i></a>
+          <a href="https://github.com/OpenDevUFCG" target="_blank"><i class="fab fa-github"></i></a>
+          <a href="https://dev.to/opendevufcg" target="_blank"><i class="fab fa-dev"></i></i></a>
+        </div>
+      </main>
+    <script src="js/counter.js"></script>
```

Comment:
Tenta botar como type [module](https://v8.dev/_img/modules/async-defer.svg)

```suggestion
    <script type="module" src="js/counter.js"></script>
```